C++/C.sublime-syntax

```diff
@@ -160,46 +167,46 @@ contexts:
         4: constant.numeric.suffix.c
         5: invalid.illegal.numeric.suffix.c
     # binary integer (C23)
-    - match: \b(0[bB])([0-1]*)(?:({{integer_suffix}})|({{bin_suffix}}))?
+    - match: \b(0[bB])([01](?:[01]|'(?=[^']))*)?(?:({{integer_suffix}})|({{bin_suffix}}))?
```

Comment:
If we want to enforce at least a digit between each `'`:

```suggestion
    - match: \b(0[bB])([01](?:'?[01])*)?(?:({{integer_suffix}})|({{bin_suffix}}))?
```

But for the sake of supporting intermediate states patterns could probably be a bit more lazy, if there are no other conflicting terms/expressions to be expected.

```suggestion
    - match: \b(0[bB])(['01]*)(?:({{integer_suffix}})|({{bin_suffix}}))?
```